spec/workers/classifications_nightly_dump_worker_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'spec_helper'
+
+RSpec.describe ClassificationsNightlyDumpWorker do
+  let(:worker) { described_class.new }
+
+  let!(:project1) { create :project }
+
+  before do
+    allow(ClassificationsDumpWorker).to receive(:perform_async)
+  end
+
+  describe "#perform" do
+    it 'enqueues a dump for all projects' do
+      from = 1.day.ago
+      till = Time.now
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.